VOM_Fortran/VOM-code/modules.f90

```diff
@@ -75,6 +75,14 @@ module vom_file_mod
       CHARACTER(len=*),parameter :: sfile_soilprofile   = 'soilprofile.par'
       CHARACTER(len=*),parameter :: sfile_model_output  = 'model_output.txt'
 
+      CHARACTER(len=*),parameter :: nfile_resultshourly = 'results_hourly.nc'
```

Comment:
This looks like it is hard-coded to produce .nc files now. The user should be able to choose if they want netcdf or the old txt output, as implied in the flag i_write_nc.

Reply:
Yes, they can by just changing the flag.

Reply:
OK, didn't see the sfile and nfile distinctions. Thanks!